tests/test-basic.sh

```diff
@@ -21,6 +21,13 @@ set -euo pipefail
 
 . $(dirname $0)/libtest.sh
 
+# This test looks for specific localized strings. Prefer a UTF-8 locale
+if locale -a | grep '^C\.UTF-8$' >/dev/null; then
```

Comment:
Do we really need this extra clutter ? Does Debian support systems without C.UTF-8 ? Even if Debian does, is there a good reason for supporting this in the flatpak test setup ?